### PR TITLE
Cloud compute node: Archive, cephfs and queue processors

### PR DESCRIPTION
**Pull request by gc-code**
### Summary of work

Ansible scripts for setting up an Autoreduction compute node. This PR contains the scripts for the archive and cephfs mounts, it also contains setup for the queue processors.

### How to test your work

Usage documentation in README.md

### CODE REVIEW

File: build/ansible-compute/group_vars/all/vars.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+development: true
+autoreduction_user: 'isisautoreduce'
+
+# Database
+db_host: '{{ vault_db_url }}'
+db_username: '{{ vault_db_root_username }}'
+db_password: '{{ vault_db_root_password }}'
+
+# Queue processors
+qp_log_file: 'queueProcessor.log'
```
Comment (ElliotAOram) [inline build/ansible-compute/group_vars/all/vars.yml]: Why is this stored in the vault?

**Comment by ElliotAOram** [comment]
Currently having an issue with adding ISIS Archive. I've messaged you on slack regarding this as it is sensitive info
```
TASK [archive : Mount the ISIS archive] ***************************************************************************************************************************************************************
fatal: FAILED! => {"changed": false, "msg": "Error mounting /isis: mount error(13): Permission denied\nRefer to the mount.cifs(8) manual page (e.g. man mount.cifs)\n"}
```
On running ansible

File: build/ansible-compute/group_vars/all/vars.yml (hunk 1 above)
Comment (gc-code) [reply build/ansible-compute/group_vars/all/vars.yml]: It isn't stored in the vault. It is just stored in the global variables file. The vault contains all the keys that look like '{{ vault_**** }}'. You can see that it takes a fixed value (queueProcessor.log), therefore isn't referencing anything in vault.yml.

Comment (ElliotAOram) [reply build/ansible-compute/group_vars/all/vars.yml]: Ah right I follow you, thanks for clarifying.

**Comment by ElliotAOram** [comment]
work with develop mode on just running it through with `develop : false`

**Comment by ElliotAOram** [comment]
@gc-code there appears to be a problem with the production setup. 
1. I have defined `development: false` in the vault.yml file
1. I run the ansible playbook using the vault
1. No errors occur
1. `/instrument` exists but is empty
1. There is no `/etc/ceph/isis.key` file on the host machine

I have tried this on a fresh host machine.

**Comment by ElliotAOram** [comment]
I have found the problem the yml file should state `development: 'false'` (WITH QUOTATIONS) for clarity I would change the `true` in vars.yml to be `'true'`

**Comment by ElliotAOram** [comment]
Merged develop in order to resolve MySQL versioning issue on Travis

**Pull request merged**